src/esp/assets/ResourceManager.cpp

```diff
@@ -1338,12 +1344,6 @@ void ResourceManager::loadTextures(Importer& importer,
       Mn::GL::TextureFormat format;
       if (image->isCompressed()) {
         format = Mn::GL::textureFormat(image->compressedFormat());
-      } else if (compressTextures_ &&
```

Comment:
Why did we remove this code fragment?

Reply:
Maybe the naming of the boolean is just bad, but we shouldn't we be able to require textures AND compress them?